bigbluebutton-html5/imports/ui/components/dropdown/list/styles.scss

```diff
@@ -17,6 +17,10 @@
   }
 }
 
+.actionsHeader {
```

Comment:
The the class for the ListTitle is called `actionsHeader`? 

Or the component needs to be changed to `DropdownListHeader` or the classname should be something like `.title`